PersonalFinanceTool.pro

```diff
@@ -0,0 +1,50 @@
+###################################################################################
+#   File name:          PersonalFinanceTool.pro
+#   Created on:         08/30/2024
+#   Author:             stwomey
+#   Description:        Project file for the PersonalFinanceTool
+#   Change History:     See https://github.com/SeanTwomey9/PersonalFinanceTool
+#   Relative Location:  PersonalFinanceTool/
+###################################################################################
+
+QT += core
+QT += gui
+
+greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
```

Comment:
Does `Qt += widgets` need to be included based on the Qt version, or should it just be included with the core and gui lines above?

Reply:
This is auto-generated in Qt `.pro` files, since the Qt version being used here is 6.7.1 widgets will be included automatically. This isn't causing any issues and indicates that the Qt version to be used should be higher than major version 4 as this application relies on widgets, so it should be ok to leave it.